### ISSUE TITLE
Allow passing new RealtimeField as string

### ISSUE THREAD

**Issue by assylias** [assylias/jBloomberg#47]
If a new realtime field is exposed by the Bloomberg API, it can't be access until the RealtimeField enum is updated.

Allow to pass a realtime field as a string to the `SubscriptionBuilder`.

**Comment by Kurry** [comment]
Yes! Both a `Collection<String> fields` and `String field`, the number of real time fields are in the hundreds if not thousands, and so I think this makes the most sense. I have to do this in my own fork. Also, adding the ability to add subscription options would be helpful. I like this change because really, you could just get rid of the `SubscriptionHolder` class. Not only should you allow people to pass new RealtimeField, you also should allow them to pass strings for subscription options.
